frontend/kick-covid-19/src/components/Answer.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 
 function Answer(props) {
+  const username = props.user ? props.user.userName : "";
   return (
     <div className="shadow p-4 bg-pink-100 hover:bg-pink-200 mt-4">
       <div className="text-left">
-        <h3 className="mb-2 text-gray-700">Card Title</h3>
+        <h3 className="mb-2 text-gray-700">{username}</h3>
         <p className="text-grey-600 text-sm">{props.body}</p>
```

Comment:
I feel like you'd want the question to be of higher importance than the user who wrote the question and I would probably switch these
```suggestion
        <h3 className="mb-2 text-gray-700">{props.body}</h3>
        <p className="text-grey-600 text-sm">{username}</p>
```